pkgs/by-name/je/jetuml/package.nix

```
@@ -0,0 +1,67 @@
+{
+  stdenvNoCC,
+  lib,
+  makeDesktopItem,
+  fetchurl,
+  jdk,
+  makeWrapper,
+}:
+
+let
+  jdkWithFX = jdk.override { enableJavaFX = true; };
+in
+stdenvNoCC.mkDerivation rec {
+
+  pname = "jetuml";
+  version = "3.9";
+
+  src = fetchurl {
+    url = "https://github.com/prmr/JetUML/releases/download/v${version}/JetUML-${version}.jar";
+    hash = "sha256-wACGbHeRQ5rXcuI1J3eTfQraWp8eWtkIAPo7BNGcFUU=";
+  };
+
+  dontUnpack = true;
+
+  buildInputs = [
+    jdkWithFX
+    makeWrapper
+  ];
```

So `buildInputs` are basically "the stuff you input into the build, that will eventually run on the target machine". Since `makeBinaryWrapper` runs on the build machine, it must be put in `nativeBuildInputs` which is "things that run on the build machine, but can make things for the target machine"

Also all `mkDerivation` stuff should use `strictDeps` now. 
```suggestion
  strictDeps = true;
  nativeBuildInputs = [
    makeBinaryWrapper
  ];
```